src/executor/stack/executor.rs

```diff
@@ -519,18 +519,12 @@ impl<'config, 'precompiles, S: StackState<'config>, P: PrecompileSet>
 		gas_limit: u64,
 		access_list: Vec<(H160, Vec<H256>)>, // See EIP-2930
 	) -> (ExitReason, Vec<u8>) {
-		if self.nonce(caller) >= U64_MAX {
```

Comment:
Just out of curiosity. Why don't we need to check nonce here?

Reply:
Because it MUST be validated inside `create_inner`.